chart/config/unsupported.yaml

```
@@ -26,3 +26,10 @@ unsupported:
     You are reusing an old and incompatible configuration, either via a values.yaml file from an
     older release, or by using the --reuse-values option. Please start with a fresh configuration,
     only changing settings relative to the defaults in the current release.
+
+  sizing.database.instances: |
+    Database replicas are not allowed.
+    High-availability is not currently feasible to support in k8s.
+    Galera replication on rolling updates of the statefulset is hard to get
+    right and to maintain as part of KubeCF. For an HA database, prefer
+    using a specialized external one.
```

Hmm, for user-facing text we should probably avoid using abbreviations.
```suggestion
  # Do not allow replicas for the database; doing this well in Kubernetes is
  # more effort than we can expend, and is better handled by external databases.
  sizing.database.instances: >
    Database replicas are not supported.
    KubeCF does not support highly available database configurations;
    please use an external database instead.
```

We don't _really_ need to say why in the error message (but putting it in a comment near the message is probably useful).  Details about Galera etc. would definitely belong more in a comment, I think.

Hmm, should probably mention [the documentation](https://kubecf.io/docs/deployment/kubernetes-deploy/#external-database).